rswag-specs/lib/rswag/specs/request_factory.rb

```diff
@@ -132,6 +132,8 @@ def add_payload(request, parameters, example)
 
         if [ 'application/x-www-form-urlencoded', 'multipart/form-data' ].include?(content_type)
           request[:payload] = build_form_payload(parameters, example)
+        elsif [ 'application/xml', 'text/xml' ].include?(content_type)
+          request[:payload] = build_form_payload(parameters, example)        
```

Comment:
Did you mean to refer to `build_xml_payload` here?
If you could add a test to express this it would be close to merging.

Reply:
Thanks! :) I'll write a test for it